extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -164,8 +164,9 @@ public function getArticleComments() {
 	private function prepareWikiVariables() {
 		$wikiVariables = $this->mercuryApi->getWikiVariables();
 		$navigation = $this->getNavigation();
-		if ( empty( $navData ) ) {
-			\Wikia\Logger\WikiaLogger::instance()->error(
+
+		if ( empty( $navigation ) ) {
+			\Wikia\Logger\WikiaLogger::instance()->debug(
 				'Fallback to empty navigation'
```

Comment:
make it notice please. It was logged as an error before our code refactor back in January. On the other hand I am wondering if we ever going to do something with this information.